cmd/main.go

```diff
@@ -139,6 +139,7 @@ func main() {
 	if err := validateConfig(conf); err != nil {
 		logs.Fatal(err)
 	}
+	conf.PublicEndpoint = strings.TrimRight(conf.PublicEndpoint, " ")
```

Comment:
Please use TrimSpace. We want all whitespace trimmed.

Reply:
I dont think space in between would pass `ParseRequestURI`. Only trailing spaces are allowed

Reply:
That's not what TrimSpace does. It remove all leading and trailing whitespace. It does not touch the middle of the endpoint.

Reply:
Sir, `ParseRequestURI` only allows trailing spaces. So we only need to remove trailing spaces

Reply:
If there's no leading spaces then TrimSpace won't remove anything. Non-issue.